feature-libs/product/configurators/rulebased/core/model/configurator.model.ts

```diff
@@ -173,6 +173,14 @@ export namespace Configurator {
     GALLERY = 'GALLERY',
   }
 
+  export enum DataType {
+    INPUT_STRING = 'String',
+    INPUT_NUMBER = 'Number',
+    QTY_ATTRIBUTE_LEVEL = 'Quantity',
+    QTY_VALUE_LEVEL = 'Attr.Quantity',
+    N_A = 'N/A',
+    NOT_IMPLEMENTED = 'not_implemented',
```

Comment:
Can't we have for example an INPUT_NUMMBER with QTY_ATTRIBUTE_LEVEL?
in my opinion that would be 2 Enums:
- STRING/NUMBER/NOT_IMPLEMENTED
- QTY_ATTRIBUTE_LEVEL/ QTY_VALUE_LEVE/NO_QUANTITY

Reply:
No. We can not have such combinations